0.2/docker-entrypoint.sh

```diff
@@ -60,15 +60,27 @@ if [ "$1" == 'janusgraph' ]; then
     fi
   done < <(env)
 
-  # wait for storage
-  if ! [ -z "${JANUS_STORAGE_TIMEOUT:-}" ]; then
-    F=$(mktemp --suffix .groovy)
-    echo "graph = JanusGraphFactory.open('${JANUS_CONFIG_DIR}/janusgraph.properties')" > $F
-    timeout ${JANUS_STORAGE_TIMEOUT}s bash -c \
-      "until bin/gremlin.sh -e $F > /dev/null 2>&1; do echo \"waiting for storage...\"; sleep 5; done"
-  fi
+  if [ "$2" == 'write-config' ]; then
+    echo "Config write into ${JANUS_CONFIG_DIR}"
```

Comment:
What do you mean by this? Is this just a note to the user as to where they should write their config? "Config write" implies that a config *will be written* but that doesn't actually happen, since this exits right after this message.

How about making the command `config-dir` and the message:

```
JanusGraph config dir: ${JANUS_CONFIG_DIR}
```

Thoughts?

Reply:
At this point the config is already written into ${JANUS_CONFIG_DIR}. My idea is to use this command for testing, if you want pre generate the config without starting janusgraph.

Reply:
Right, I understand; what I mean is the command `write-config` sounds very similar to `print-config` — in isolation, they sound very similar to the point of being identical/confusing.

The first one just prints the directory where you may want to run the config, the second one actually shows the config, which is why I'm suggesting changing the name of `write-config` to just talk about the directory.

Thoughts?

Reply:
I'm okay with just having a single command something like `generate-and-print-config`.

Reply:
Sure, sounds good to me — how about something shorter, e.g., `show-config` or `print-config` or similar, which prints out both the directory as well as the two config files?

Reply:
I think `show-config` is good.

Reply:
SGTM, thanks!